app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -936,7 +936,9 @@ class AdvancedFilter extends React.Component {
    * @param {String} additionalFilterOption - Selected option from additional list of options (if provided)
    */
   renderNumberStatement = (filter, index, value, numberOption, additionalFilterOption) => {
-    const betweenTooltipText = '"Between" is inclusive and will filter for values within the user-entered range, including the start and end values.';
+    const numberOptionTooltipText = 'Leaving number field blank will not filter out any monitorees.';
```

Comment:
thoughts on tacking this onto the end of the betweenTooltipText conditionally so there is one variable?  more just to male this cleaner so there is only `renderTooltipStatement` needed

Reply:
you could have a conditional to determine text for the tooltip... should there be one for the blank numberOption too explaining what that is searching for?

Reply:
I actually thought about these 2 things when working on this, will make those changes then

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1252/commits/1f615027510e14bc63773a1044e97dcb3f2af524